t/01_ps.t

```diff
@@ -1,18 +1,17 @@
 use strict;
 use warnings;
+use FindBin;
 use File::Spec;    # try to keep pathnames neutral
 use Test::More 0.96;
 
-BEGIN { use_ok("HTML::FormatPS"); }
+use lib "$FindBin::Bin/lib";
```

Comment:
it's still an unnecessary complication to a test.  FindBin is only really useful for relocatable scripts, outside of the cpan context.